requirements/dev-requirements.txt

```diff
@@ -15,7 +15,7 @@ fixture
 sniffer
 psutil  # for memory profiling
 django-extensions
-ipython==4.2.1
+ipython==5.2.2
```

Comment:
yes! nice that was the error i was getting. auto complete also wasn't working unless i upgraded ipdb